server/util/webscraping/scheduleScrape.js

```diff
@@ -1,26 +1,72 @@
 const webScrape = require('./webScrape');
+const getCoordsForAddress = require('../apis/location');
 const schedule = require('node-schedule');
+const mapUtil = require('../map');
 
 async function scrapeAll() {
 	// Providence
-	webScrape(
+	let providence = await webScrape(
 		'https://data.providenceri.gov/Public-Safety/Providence-Police-Case-Log-Past-180-days/rz3y-pz8v',
 		'.dataset-preview'
 	);
+	let crimesProvidencce = providence.map((doc) => {
+		if (doc.length != 0) {
+			let coordinates = getCoordsForAddress(doc[1] + ', Providence');
+			return {
+				type: doc[5],
+				address: doc[1],
+				date: doc[2],
+				location: {
+					longitude: coordinates.lng,
+					latitude: coordinates.lat,
+				},
+			};
+		}
+	});
 
-	// Chicago
-	webScrape(
+	crimesProvidencce = crimesProvidencce.filter((crime) => crime !== undefined);
+	//return crimesProvidencce;
+	//save to db
+	let providenceDB = await mapUtil.createCrime(crimesProvidencce);
+
+	// // Chicago
+	let chicago = await webScrape(
 		'https://data.cityofchicago.org/Public-Safety/Crimes-2023/xguy-4ndq',
 		'.dataset-preview'
 	);
+	let crimesChicago = chicago.map((doc) => {
+		if (doc.length !== 0) {
+			let coordinates = getCoordsForAddress(doc[3].substring(6) + ', Chicago');
+			return {
+				type: doc[5],
+				address: doc[3].substring(6),
+				date: doc[2],
+				location: {
+					longitude: coordinates.lng,
+					latitude: coordinates.lat,
+				},
+			};
+		}
+	});
```

Comment:
Can we make a reusable function somehow? I notice this overlaps with the crimes in Providence for the most part